code/game/gamemodes/objective_items.dm

```diff
@@ -400,3 +440,79 @@
 
 /obj/item/blackbox/add_stealing_item_objective()
 	return add_item_to_steal(src, /obj/item/blackbox)
+
+
+// A number of special early-game steal objectives intended to be used with the steal-and-destroy objective.
+// They're basically items of utility or emotional value that may be found on many players or lying around the station.
+/datum/objective_item/steal/traitor/insuls
+	name = "insulated gloves"
+	targetitem = /obj/item/clothing/gloves/color/yellow
+	excludefromjob = list(JOB_CARGO_TECHNICIAN, JOB_QUARTERMASTER, JOB_ATMOSPHERIC_TECHNICIAN, JOB_STATION_ENGINEER, JOB_CHIEF_ENGINEER)
+	item_owner = list(JOB_STATION_ENGINEER, JOB_CHIEF_ENGINEER)
+	exists_on_map = TRUE
+
+/obj/item/clothing/gloves/color/yellow/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/clothing/gloves/color/yellow)
+
+/datum/objective_item/steal/traitor/moth_plush
+	name = "cute moth plush toy"
+	targetitem = /obj/item/toy/plush/moth
+	excludefromjob = list(JOB_PSYCHOLOGIST, JOB_PARAMEDIC, JOB_CHEMIST, JOB_MEDICAL_DOCTOR, JOB_VIROLOGIST, JOB_CHIEF_MEDICAL_OFFICER, JOB_CORONER)
+	exists_on_map = TRUE
+
+/obj/item/toy/plush/moth/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/toy/plush/moth)
+
+/datum/objective_item/steal/traitor/lizard_plush
+	name = "cute lizard plush toy"
+	targetitem = /obj/item/toy/plush/lizard_plushie
+	exists_on_map = TRUE
+
+/obj/item/toy/plush/lizard_plushie/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/toy/plush/lizard_plushie)
+
+/datum/objective_item/steal/traitor/denied_stamp
+	name = "cargo's denied stamp"
+	targetitem = /obj/item/stamp/denied
+	excludefromjob = list(JOB_CARGO_TECHNICIAN, JOB_QUARTERMASTER, JOB_SHAFT_MINER)
+	exists_on_map = TRUE
+
+/obj/item/stamp/denied/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/stamp/denied)
+
+/datum/objective_item/steal/traitor/granted_stamp
+	name = "cargo's granted stamp"
+	targetitem = /obj/item/stamp/granted
+	excludefromjob = list(JOB_CARGO_TECHNICIAN, JOB_QUARTERMASTER, JOB_SHAFT_MINER)
+	exists_on_map = TRUE
+
+/obj/item/stamp/granted/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/stamp/granted)
```

Comment:
_Little_ concerned about how this one will shake out: I feel like it will often just offer you both items at once because they have the same requirements, and as they are often both left in the same place that seems a bit silly. Stealing the "granted" stamp is probably more annoying than the denied one.

Reply:
It just seemed like a decent duo mix that may have people breaking into cargo. If they get lucky with both stamps at the same time I think they can have it. The rep/TC reward is low (the same as posters and graffiti) and often these starting objectives don't even give any TC, just rep.